modules/serverless-enabling-pvc-support.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: REFERENCE
+[id="serverless-enabling-pvc-support_{context}"]
+= Enabling PVC support
+
+Some serverless applications need permanent data storage. To achieve this, you can configure Persistent Volume Claims (PVCs) for your Knative services.
+
+* The `"kubernetes.podspec-persistent-volume-claim"` extension controls whether Persistent Volumes (PVs) can be used with Knative Serving.
+* The `"kubernetes.podspec-persistent-volume-write"` extension controls whether PVs are available to Knative Serving with the write access.
+
+To enable Knative Serving to use PVCs and write to them, modify the `KnativeServing` custom resource (CR) to include the following YAML:
+
+.Enabling PVCs with write access
+[source,yaml]
+----
+...
+spec:
+  config:
+    features:
+      "kubernetes.podspec-persistent-volume-claim": enabled
+      "kubernetes.podspec-persistent-volume-write": enabled
+...
+----
+
+To claim a PV, modify your service to include the PV configuration. For example, the following YAML claims a PV with write access:
+
+.PersistentVolumeClaim configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: example-pv-claim
+  namespace: my-ns
+spec:
```

Comment:
Users should use a storage class here as well that supports the specific access mode. For example if users run on gcloud and use the default one which is `gp2` the pvc will never be ready as it does not support ReadWriteMany.
So let's add that as a note and the field: `storageClassName`:  ...`.